src/cpp/core/r_util/RActiveSessionStorage.cpp

```diff
@@ -158,7 +158,7 @@ namespace
 
     FilePath FileActiveSessionStorage::getPropertyDir(const std::string& id) const
     {
-        FilePath propertiesDir = activeSessionsDir_.completeChildPath(fileSessionDirPrefix_ + id + "/" + propertiesDirName_);
+        FilePath propertiesDir = scratchPath_.completeChildPath(propertiesDirName_);
```

Comment:
Does the `id` variable not need to be used here somewhere?

Reply:
The id variable does not need to be used, as the FileActiveSessionStorage constructor takes in the scratchPath, which is the session/active/session-<id> path. That is the ID is already baked in at initialization time. We only do the getPropertyDir to grab the `properites` path because presumabley in the future we'll be getting more types of session data than just the properties.

This does point out that the parameter isn't being used, so I'll remove that.

Reply:
When is the ID baked in? I also see the parameter is still being passed in the getPropertyFile method but it's now unused there as well

Reply:
The ActiveSession constructor has the ID and scratchPath, and when it calls the storage factory getFileActiveSessionStorage(<FilePath>) it passes the scratchPath for the session which is passed along to the FileActiveSessionStorage constructor. The scratch path is `session/active/session-<session-id>/` path. By constructing with the session specific path from the beginning, the ID isn't needed when we're constructing the properties path.

The ID also isn't necessary for the read/writeProperties methods, but it will be needed for later DB connections, so i decided it could stay.